tools/query/query.xml

```diff
@@ -0,0 +1,134 @@
+<tool id="query" name="Query" version="@TOOL_VERSION@">
+    <macros>
+        <token name="@TOOL_VERSION@">0.1</token>
+        <xml name="formats">
+            <option value="csv">csv</option>
+            <option value="tsv">tsv</option>
+            <option value="h5">h5</option>
+            <option value="feather">feather</option>
+            <option value="parquet">parquet</option>
+            <option value="sqlite">sqlite</option>
+        </xml>
+        <xml name="internal_name" token_format="">
+            <when value="@FORMAT@">
+                <param name="name" type="text" help="Specify the format's internal name of the table."/>
+            </when>
+        </xml>
+    </macros>
+
+    <requirements>
+        <requirement type="package">click</requirement>
```

Comment:
the requirements should be versioned for many reasons including reproducibility

Reply:
The code looks actually small enough to inline the entire script as a configfile. This would reduce the dependencies and also the code in general as we do not need to pass parameters.